app/src/main/java/org/jellyfin/androidtv/ui/playback/overlay/action/ZoomAction.kt

```diff
@@ -28,39 +33,41 @@ class ZoomAction(
 		videoPlayerAdapter.leanbackOverlayFragment.setFading(false)
 		return PopupMenu(context, view, Gravity.END).apply {
```

Comment:
If you migrate this code to use our MenuBuilder you don't need to convert the enum members to int and back.

Reply:
I was looking for something like this when creating the enum, I must have missed it previously. Thanks for pointing me in the right direction!